Makefile

```diff
@@ -0,0 +1,18 @@
+
+start:
+	docker compose up -d
+
+stop:
+	docker compose down
+
+restart:
+	docker compose restart
+
+build:
+	docker compose build
+
+buildup:
+	docker compose up -d --build
+
+cleanv:
+	docker compose down -v
```

Comment:
(Copied from the email I just sent since this is better context for it --)

How about if I [some time soon, but not in this branch right now] add `make release` to cut a new (tagged, versioned) release, and `make deploy` to create a production tarball then scp it to the server?